docs/examples/general/data_loading/webdataset.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<p>"matches another one already implemented" -&gt; "matches an equivalent pipeline reading the files from an untarred directory, with <code>fn.readers.file</code>  reader"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>